lib/orogen/templates/deployment.pc

```diff
@@ -8,6 +8,9 @@ project_name=<%= project.name %>
 deffile=${prefix}/share/orogen/<%= File.basename(project.deffile) %>
 typekits=<%= project.name if project.typekit %> <%= deployer.used_typekits.map { |tk| tk.name }.join(" ") %>
 deployed_tasks=<%= deployer.task_activities.map(&:name).sort.join(",") %>
+deployed_tasks2=<%= deployer.task_activities.
```

Comment:
I would have called this `deployed_tasks_pair`

Reply:
Thanks for being my conscience sometimes ... Renamed to `deployed_tasks_with_models`.